react exercise/users-information/src/components/creator-card/CreatorCard.tsx

```diff
@@ -0,0 +1,38 @@
+import { User } from "../../types/user.ts"; // Import defaultUser
+import "./CreatorCard.scss";
+import defaultProfilePicture from "../../assets/default-profile-picture.svg";
+
+interface CreatorCardProps {
+  creatorData: User;
+}
+function CreatorCard({ creatorData }: CreatorCardProps): JSX.Element {
+  return (
+    <div className="creator-card">
+      <div className="creator-posts-container">
```

Comment:
I'm pretty sure the goal was that we will display a preview for the user posts in the gray triangles. Because this is what comes to mind and I don't see why to put gray triangles in this manner. But since the data that we have been given is lacking more than one image @roni left this logic out and so do I.